arch/x86/x86_64/include/uk/asm/arch.h

```diff
@@ -113,3 +121,80 @@
 #define X86_EFER_TCE		(1 << 15)
 
 #endif /* __PLAT_CMN_X86_CPU_DEFS_H__ */
+
+/* registers from apic_defs.h */
+#ifndef __PLAT_CMN_X86_APIC_DEFS_H__
+#define __PLAT_CMN_X86_APIC_DEFS_H__
```

Comment:
I would move the APIC definitions to `plat/common/include/x86/apic.h` instead. Also, take note of the guards
`__PLAT_CMN_X86_APIC_DEFS_H__`, they should be aligned to where the headers actually are and now you basically have both `__PLAT_CMN_X86_APIC_DEFS_H__` and `__PLAT_CMN_X86_CPU_DEFS_H__`, so it cannot be right.